packages/enclave-contracts/contracts/registry/NaiveRegistryFilter.sol

```diff
@@ -85,14 +89,13 @@ contract NaiveRegistryFilter is IRegistryFilter, OwnableUpgradeable {
 
     function publishCommittee(
         uint256 e3Id,
-        address[] calldata nodes,
-        bytes calldata publicKey
+        address[] memory nodes,
```

Comment:
why change these back to memory?

Reply:
ahh, it got copied over from the previous branch, it wasnt up to date with `main`